cloudpickle/cloudpickle.py

```diff
@@ -123,6 +127,32 @@ def _lookup_class_or_track(class_tracker_id, class_def):
     return class_def
 
 
+def register_pickle_by_value(module):
+    """Register that the input module should be pickled by value."""
+    module_name = module.__name__ if inspect.ismodule(module) else module
+    _PICKLE_BY_VALUE_MODULES.add(module_name)
+
+
+def unregister_pickle_by_value(module):
+    """ nregister that the input module should be pickled by value."""
+    module_name = module.__name__ if inspect.ismodule(module) else module
+    _PICKLE_BY_VALUE_MODULES.remove(module_name)
+
+
+def _is_explicitly_serialized_module(module, submodules=True):
```

Comment:
I think (also discussed with @ogrisel last week) that this function could also follow the `pickle_by_value` convention.
How about `is_registered_pickle_by_value`?